.github/workflows/scripts/compare_translations.py

```
@@ -0,0 +1,236 @@
+
+def load_translation(filepath):
+    """Load translation from a file.
+
+    Args:
+        filepath: Path to the translation file
+
+    Returns:
+        translation: Loaded translation
+    """
+    try:
+        with open(filepath, "r", encoding="utf-8") as file:
```

Ruff (0.14.8)

Unnecessary mode argument

Remove mode argument

(UP015)